coalib/parsing/DefaultArgParser.py

```diff
@@ -119,6 +120,26 @@ def default_arg_parser(formatter_class=None):
                                 metavar='BOOL',
                                 help='Write the logs as json to a file '
                                 'where filename is specified as argument.')
+    if parser_type == 'coala-html':
```

Comment:
I have not tried running it locally (sorry, Im on windows and I cant code here)

And even if its a dependency ... you shouldnt have bits of the code here that are specifically for coala-html.

Why not import DefaultArgParser in coala-html and add a few args there ? Seems to make sense to me